ark-cli/src/commands/storage/list.rs

```diff
@@ -0,0 +1,51 @@
+use std::path::PathBuf;
+
+use crate::{
+    models::storage::Storage, models::storage::StorageType, translate_storage,
+    AppError,
+};
+
+#[derive(Clone, Debug, clap::Args)]
+#[clap(name = "list", about = "List resources in a storage")]
+pub struct List {
+    #[clap(value_parser, help = "Root directory of the ark managed folder")]
+    root_dir: Option<PathBuf>,
+    #[clap(help = "Storage name")]
+    storage: Option<String>,
+    #[clap(short, long, action = clap::ArgAction::SetTrue, help = "Whether or not to use atomatic versioning")]
```

Comment:
It's probably more like unfolding the versions in the table.. We should discuss this feature.